scripts/ci/upload-build

```diff
@@ -2,35 +2,34 @@
 set -ex
 
 # configuration
+AWS_BUCKET='downloads.mesosphere.io'
+FILENAME='dcos-ui'
 BUILD_VARIANT=${BUILD_VARIANT:-''}
 BRANCH_NAME=${BRANCH_NAME:-'master'}
-DIST_PATH=${DIST_PATH:-'dist'}
-PACKAGE_PATH=${PACKAGE_PATH:-'.'}
-SHA_SLUG=${SHA_SLUG:-$(git rev-parse --short HEAD)}
+PKG_VERSION=$(git tag -l --contains HEAD)
 
-# aws settings
-AWS_BUCKET='downloads.mesosphere.io'
-BUCKET_PATH='dcos-ui'
+if [[ $PKG_VERSION ]]
+then
+  PKG_VERSION="-$PKG_VERSION"
+fi
 
-# build release filename
-FILENAME='dcos-ui'
-PKG_VERSION=$(python -c "import sys, json; \
-  pkg = json.load(open('$PACKAGE_PATH/package.json')); \
-  sys.stdout.write(str(pkg['version']));")
 PKG_BRANCH=$(echo ${BRANCH_NAME} | sed -E 's/([^/]*)$|./\1/g')
-RELEASE_NAME="${PKG_BRANCH}+${FILENAME}${BUILD_VARIANT}-v${PKG_VERSION}+${SHA_SLUG}.tar.gz"
+
+# ex. master+dcos-ui-v1_12_0-rc_1.tar.gz
```

Comment:
❤️

Reply:
this isnt a valid semver version number :(

Reply:
this isnt a valid semver version number :(

Reply:
This will be the tag name. So It will be valid! 💯

Reply:
I'll update the comment to be accurate.